libs/designsystem/src/lib/components/page/page.component.ts

```diff
@@ -116,12 +119,29 @@ export class PageComponent
   @Input() hideBackButton: boolean;
   @Input() titleMaxLines: number;
 
+  private _hideTabs: boolean;
+  public get hideTabs(): boolean {
+    return this._hideTabs;
+  }
+  @Input()
+  public set hideTabs(hideTabs: boolean) {
```

Comment:
Consider naming, as imperative form sounds like a method:
```suggestion
  public set tabsHidden(hide: boolean) {
```